provision-team/svcbusclient/Program.cs

```diff
@@ -68,17 +69,12 @@ static async Task SendMessagesAsync(string recipientEmail, string messageBody)
                         { "Message", messageBody }
                     };
 
-                    var serializer = DataContractBinarySerializer<Dictionary<string, string>>.Instance;
-                    byte[] data;
+                    string data = JsonConvert.SerializeObject(message);
```

Comment:
Not needed

Reply:
@JoshLove-msft, I have updated, please review.